tests/post-command.bats.ignore

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
```

Comment:
What's the `.ignore` suffix in the filename for?

Reply:
Ignoring the file 🤦 I'll fix. Thanks